srcpkgs/libclc/template

```diff
@@ -6,9 +6,8 @@ build_style=cmake
 # disable clspv (failing tests, we don't ship it)
 configure_args="-DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=Release
  -DLIBCLC_TARGETS_TO_BUILD=amdgcn--;amdgcn--amdhsa;r600--;nvptx--;nvptx64--;nvptx--nvidiacl;nvptx64--nvidiacl;spirv-mesa3d-;spirv64-mesa3d-"
-hostmakedepends="clang17 llvm17-devel python3 libedit-devel libffi-devel
+hostmakedepends="llvm17-devel python3 libedit-devel libffi-devel
  ncurses-devel zlib-devel SPIRV-LLVM-Translator"
-makedepends="llvm17-devel"
```

Comment:
only needs hostmakedepends since this template only generates header files and bitcode files